docs/user/set-up/quick-starts.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases:
+- /docs/agent/latest/set-up/quick-starts/
+title: Grafana Agent quick starts
+weight: 150
+---
+## Grafana Agent quick starts
```

Comment:
Are we calling these quick starts vs. getting started? Looks like you are referring folks to getting started guides. Should we pick one term and stick with that?

Reply:
Happy with either approach-I copied what Cloud is using, which is why I opted for quick start -- and it takes users to documentation outside of the "core Agent". Would love to discuss options for this and standardize!